release_notes.md

```diff
@@ -1,118 +1,57 @@
-    - `outbreak-threat-hunt-tools`
-    - `outbreak-threat-hunt-tools-params`
-    - `outbreak-auto-install-time-frame`
-
-- Effortlessly manage your threat hunt integrations. Each integration and its precise hunt parameters are stored in the `outbreak-threat-hunting-workflow-config` key store record, ensuring a streamlined and efficient threat-hunting process.
-
-- Enjoy seamless automation with the installation of outbreak-specific response solution packs. All associated parameters are stored in the `outbreak-alert-config` key store record, providing an easy setup for optimal protection.
-
->[!NOTE]
-> The `outbreak-threat-hunting-workflow-config` and the `outbreak-alert-config` key store records are created *after* the Outbreak Response Framework configuration completes.
-
-#### Playbook Enhancements
-
-- The following hunt playbooks are created within SOAR Framework's **05 - Hunt** playbook collection to consolidate all hunt-related playbooks in one location. This reorganization simplifies navigation and management by keeping all hunt playbooks together, which streamlines processes and reduces confusion:
-
-    - IOC Hunting
-    - IOC Hunting – FortiSIEM
-    - IOC Hunting - FortiSIEM > Fetch Incident Associated Events
-    - IOC Hunting - FortiAnalyzer
-    - IOC Hunting - FortiAnalyzer > Device Log Search
-    - IOC Hunting - Splunk
-    - IOC Hunting - IBM QRadar
-    - Threat Hunting (Fortinet Fabric) - FortiSIEM
-    - Threat Hunting (Fortinet Fabric) - FortiSIEM > Fetch Incident
-    - Threat Hunting (Fortinet Fabric) - FortiSIEM > Create Incident
-    - Threat Hunting (Fortinet Fabric) - FortiSIEM > Fetch Associated Incident Events
-    - Threat Hunting (Fortinet Fabric) - FortiAnalyzer
-    - Threat Hunting (Fortinet Fabric) - FortiAnalyzer > Get Related Event Assets
-    - Threat Hunting (Signature Based) - Splunk
-    - Threat Hunting (Signature Based) - Splunk > Create Inbound Incident
-    - Threat Hunting (Signature Based) - Splunk > Update Notable Fields
-    - Threat Hunting (Signature Based) - IBM QRadar
-    - Threat Hunting (Signature Based) - Get Events - FortiSIEM
-    - Threat Hunting (Signature Based) - Get Events - FortiAnalyzer
-    - Threat Hunting (Signature Based)  - FortiAnalyzer > Log Search
-
-#### Miscellaneous Updates
-
-We have streamlined our integrations by removing the following connectors from the Outbreak Response Framework:
-
-- *Fortinet FortiGate*: Playbooks are now natively available in the SOAR Framework solution pack.
-- *Azure Log Analytics*: Usage data indicates minimal engagement.
-- *Jira* and *ServiceNow*: Integration no longer contributes to investigative processes.
-
-The Ticketing/ITSM integration page has been removed from the Configuration Wizard.
\ No newline at end of file
+- The step *Investigate Outbreak Alert* has been renamed to **Ingest Outbreak CVEs and IOCs Details** in the *Retrieve CVEs and IOCs* playbook to better convey its functionality
```

Comment:
IMO, no need to mention this point as it will not impact any functionality.

Reply:
It is a change, nonetheless. Let's keep it.